vsphere/datadog_checks/vsphere/data/conf.yaml.example

```diff
@@ -113,13 +106,20 @@ instances:
     #
     # collection_level: 1
 
-    ## @param collect_realtime_only - boolean - optional - default: true
-    ## Set to true to collect only host and VM metrics. Set to false to collect
-    ## cluster, datastore and datacenter metrics as well.
-    ## Collecting non realtime metrics will decrease performance as it
-    ## requires computation from the vCenter server.
+    ## @param collect_historical_only - boolean - optional - default: false
+    ## By default the check collects metrics from realtime resources (hosts and VMs).
+    ## Set this to true to only collect metrics from historical resources
+    ## (cluster, datastore and datacenter).
+    ## In order to collect metrics from those two types of resources, create two
+    ## 'instances' in this config file pointing to the same vcenter server, one with this flag
+    ## to false, the other with this flag to true.
+    ## The reason for this requirement is that collecting historical resources decreases
+    ## the performance of the check; having two instances allows realtime metrics collection
+    ## not to be slowed down by the collection of historical metrics.
+    ## Note: An instance with this flag set to true does not submit any service check, events or host tags.
+    ##       It is designed to be used only when next to another instance of the check.
```

Comment:
IIUC, you want to phase out the check that reports both at the same time? Are we sure it doesn't make sense for "small" deployments?

Reply:
Just as an example, our test environment, the check takes 5 times longer to complete because of those historical metrics.

My point is that it is always better to collect historical metrics in a separate instance and this is what I'm enforcing here (note that the previous behavior with `collect_realtime_only` is still supported but deprecated)